packages/nameguard-react/src/components/Report/CheckResultCodeIcon.tsx

```diff
@@ -28,6 +28,7 @@ function text(code: CheckResultCode | Rating) {
 function icon(
   code: CheckResultCode | Rating,
```

Comment:
Let's please ensure we remove any code that doesn't strictly separate `CheckResultCode` from `Rating`. These are never to be treated like they are the same thing.